diff --git a/host/libraries/libbladeRF/CMakeLists.txt b/host/libraries/libbladeRF/CMakeLists.txt
index 095df5b..a80dc40 100644
--- a/host/libraries/libbladeRF/CMakeLists.txt
+++ b/host/libraries/libbladeRF/CMakeLists.txt
@@ -41,6 +41,9 @@ if (NOT EXISTS "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}")
     message(STATUS "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR} does not exist. Defaulting libbladeRF install location to ${CMAKE_INSTALL_PREFIX}/lib.")
     set(CMAKE_INSTALL_LIBDIR lib)
 endif()
+# This logic is flawed for cross-compiling using OE.  package libdirs
+# won't exist at config time. Nuking for now.
+set(CMAKE_INSTALL_LIBDIR lib64)
 
 ################################################################################
 # Configuration options
