diff --git a/README.TODO-FindGnurdio.cmake-missing b/README.TODO-FindGnurdio.cmake-missing
deleted file mode 100644
index 6be78e7..0000000
--- a/README.TODO-FindGnurdio.cmake-missing
+++ /dev/null
@@ -1,3 +0,0 @@
-By not providing "FindGnuradio.cmake" in CMAKE_MODULE_PATH this project has
-|   asked CMake to find a package configuration file provided by "Gnuradio",
-|   but CMake did not find one.
